src/main/php/PDepend/Metrics/Analyzer/ClassDependencyAnalyzer.php

```diff
@@ -68,22 +68,7 @@ class ClassDependencyAnalyzer extends AbstractAnalyzer
     /**
      * Hash with all calculated node metrics.
      *
-     * <code>
-     * array(
-     *     '0375e305-885a-4e91-8b5c-e25bda005438'  =>  array(
-     *         'loc'    =>  42,
-     *         'ncloc'  =>  17,
-     *         'cc'     =>  12
-     *     ),
-     *     'e60c22f0-1a63-4c40-893e-ed3b35b84d0b'  =>  array(
-     *         'loc'    =>  42,
-     *         'ncloc'  =>  17,
-     *         'cc'     =>  12
-     *     )
-     * )
-     * </code>
-     *
-     * @var array<string, array>
+     * @var array<string, array<string, array<string, int>>>
```

Comment:
Here I would keep both, the type and the example.

Also it's possible to use an array shape:
```suggestion
     * @var array<string, array<string, array{
     *   loc: int;
     *   ncloc: int;
     *   cc: int;
     * }>>
```

Reply:
but the example is wrong, its coped from another class and does not match how this one works. its probably a leftover from when this class was originally adapted from another one.

Reply:
Regarding array shapes, I don't know how well it's supported by other tools, but we could apply if you think it's a good idea.